packages/components/src/components/Dropdown/index.tsx

```diff
@@ -127,12 +217,23 @@ interface GroupedMenuItems {
 interface MenuItemProps {
     item: DropdownMenuItem;
 }
+
+interface MasterLink {
+    label: React.ReactNode;
+    icon: IconProps['icon'];
+    callback?: () => any | Promise<any>;
```

Comment:
minor: 
The result of the callback is not used and promise is not awaited so I would prefer to keep it strict and do not allow any return value from callback.
```suggestion
    callback?: () => void;
```